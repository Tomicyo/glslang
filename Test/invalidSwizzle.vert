#version 420

void f();

void main() {
    f().xx; // Scalar swizzle does not apply to void
    f().xy; // Vector swizzle does not apply either
}