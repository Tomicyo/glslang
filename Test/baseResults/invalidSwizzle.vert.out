invalidSwizzle.vert
ERROR: 0:6: 'xx' : does not apply to this type:  global void
ERROR: 0:7: 'xy' : does not apply to this type:  global void
ERROR: 2 compilation errors.  No code generated.


Shader version: 420
ERROR: node is still EOpNull!
0:5  Function Definition: main( ( global void)
0:5    Function Parameters: 
0:6    Sequence
0:6      Function Call: f( ( global void)
0:7      Function Call: f( ( global void)
0:?   Linker Objects
0:?     'gl_VertexID' ( gl_VertexId int VertexId)
0:?     'gl_InstanceID' ( gl_InstanceId int InstanceId)


Linked vertex stage:

ERROR: Linking vertex stage: No function definition (body) found: 
    f(

Shader version: 420
ERROR: node is still EOpNull!
0:5  Function Definition: main( ( global void)
0:5    Function Parameters: 
0:6    Sequence
0:6      Function Call: f( ( global void)
0:7      Function Call: f( ( global void)
0:?   Linker Objects
0:?     'gl_VertexID' ( gl_VertexId int VertexId)
0:?     'gl_InstanceID' ( gl_InstanceId int InstanceId)

